src/SFA.DAS.LevyTransferMatching.Api/Controllers/PledgeController.cs

```diff
@@ -247,7 +265,20 @@ public async Task<IActionResult> PledgeApplications(int pledgeId)
                 PledgeStatus = queryResult?.PledgeStatus
             });
         }
-        
+
+        [Authorize(Policy = PolicyNames.PledgeAccess)]
+        [HttpGet]
+        [Route("accounts/{accountId}/pledges/{pledgeId}/applications-account-names")]
```

Comment:
The pattern is for endpoints in the outer api to have matching routes with the page in the UI which they serve, except that values are not encoded.

Since this page is there to support the `.../applications/reject-applications` `GET` endpoint in the UI, I'd expect it to also have a route of `.../applications/reject-applications`, and for the method, query, handler etc. names to reflect that.

This pattern makes it possible to see which endpoints serve which pages really quickly, and to discourage inappropriate re-use of outer api endpoints.

Reply:
Yes sure. I can change that. The only reason I created this with that name is so that this endpoint only returns a list of the DAS Account Names. I will change it to reject-applications (GET) and will also change the related methods to reflect this change.

Reply:
I have changed the GET endpoint and method, query, handler as well.